kb/temporal-platform-limits-sheet.md

```diff
@@ -64,3 +66,8 @@ Hard limits fail with an error; soft limits produce a warning log on the server
     - keyword: 20
     - text: 5
   - Retention Period: 30 days (configurable; range of 1–90 days)
+- **List Filters**
+  - The **ORDER BY** operator is not supported in List Filters in Temporal Cloud.
+    This means that you cannot apply custom ordering of Workflows with Cloud Visibility features.
+    Lists of Workflows are still ordered by a default ordering rule.
```

Comment:
Should we specify what's the order?

Reply:
@rodrigozhou I don't think it is described anywhere. But I also understand that it is subject to change at any time... which is called in below.
Shall we cap it here and add it if it is seems relevant to do so in the future?

Reply:
If it's not described anywhere, I think it's fine to not to say it here.
Not sure if it's good to mention the specific order or not given that it's subject to change.